tensorflow_serving/test_util/test_util.cc

```diff
@@ -16,16 +16,25 @@ limitations under the License.
 #include "tensorflow_serving/test_util/test_util.h"
 
 #include "tensorflow/core/lib/io/path.h"
+#include "tensorflow/core/platform/env.h"
 #include "tensorflow/core/platform/test.h"
 
 namespace tensorflow {
 namespace serving {
 namespace test_util {
 
 string TestSrcDirPath(const string& relative_path) {
+  string base_path = tensorflow::io::JoinPath(
+      getenv("TEST_SRCDIR"),
+      "org_tensorflow_serving/tensorflow_serving");
+  if (Env::Default()->FileExists(base_path)) {
+    // Supported in Bazel 0.2.2+.
+    return tensorflow::io::JoinPath(base_path, relative_path);
+  }
+  // Old versions of Bazel sometimes don't include the workspace name in the
+  // runfiles path.
   return tensorflow::io::JoinPath(
-      tensorflow::io::JoinPath(getenv("TEST_SRCDIR"),
-                               "tensorflow_serving"),
+      tensorflow::io::JoinPath(getenv("TEST_SRCDIR"), "tensorflow_serving"),
```

Comment:
Please move "tensorflow_serving" to the next line right below getenv like in the original.